### PR TITLE
mv: Show 'skipped' when a file isn't overwriten

### PR DESCRIPTION
**Pull request by sylvestre**
Should fix tests/mv/mv-n.sh

### CODE REVIEW

**Comment by sylvestre** [comment]
@cakebaker @tertsdiepraam 
Any idea what is going on?

```
--- TRY 3 STDERR:        coreutils::tests test_mv::test_mv_arg_interactive_skipped_vin ---
thread 'test_mv::test_mv_arg_interactive_skipped_vin' panicked at 'called `Result::unwrap()` on an `Err` value: Custom { kind: Other, error: "failed to write to stdin of child: Broken pipe (os error 32)" }', tests/common/util.rs:2004:18
stack backtrace:
   0: rust_begin_unwind
             at /rustc/84c898d65adf2f39a5a98507f1fe0ce10a2b8dbc/library/std/src/panicking.rs:579:5
   1: core::panicking::panic_fmt
             at /rustc/84c898d65adf2f39a5a98507f1fe0ce10a2b8dbc/library/core/src/panicking.rs:64:14
   2: core::result::unwrap_failed
             at /rustc/84c898d65adf2f39a5a98507f1fe0ce10a2b8dbc/library/core/src/result.rs:1750:5
   3: core::result::Result<T,E>::unwrap
             at /rustc/84c898d65adf2f39a5a98507f1fe0ce10a2b8dbc/library/core/src/result.rs:1090:23
   4: tests::common::util::UChild::wait_with_output
             at ./tests/common/util.rs:2001:13
   5: tests::common::util::UChild::wait
             at ./tests/common/util.rs:1948:22
   6: tests::common::util::UCommand::run
             at ./tests/common/util.rs:1543:9
   7: tests::common::util::UCommand::fails
             at ./tests/common/util.rs:1567:26
   8: tests::test_mv::test_mv_arg_interactive_skipped_vin
             at ./tests/by-util/test_mv.rs:1263:5
   9: tests::test_mv::test_mv_arg_interactive_skipped_vin::{{closure}}
             at ./tests/by-util/test_mv.rs:1259:42
  10: core::ops::function::FnOnce::call_once
             at /rustc/84c898d65adf2f39a5a98507f1fe0ce10a2b8dbc/library/core/src/ops/function.rs:250:5
  11: core::ops::function::FnOnce::call_once
             at /rustc/84c898d65adf2f39a5a98507f1fe0ce10a2b8dbc/library/core/src/ops/function.rs:250:5
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

works locally

**Comment by cakebaker** [comment]
Hm, the code looks fine and works locally, too. Maybe it's the unnecessary `pipe_in` call?

**Comment by sylvestre** [comment]
I used ignore_stdin_write_error to ignore the stdin

**Comment by tertsdiepraam** [comment]
> Any idea what is going on?

Not really, but I have some vague recollection of this happening due to nextest in some other PR?

File: tests/by-util/test_mv.rs [hunk 1]
```diff
@@ -1243,6 +1243,31 @@ fn test_mv_info_self() {
...
+        .stdout_is("skipped 'b'\n");
+}
+
+#[test]
+fn test_mv_arg_interactive_skipped_vin() {
+    let (at, mut ucmd) = at_and_ucmd!();
+    at.touch("a");
+    at.touch("b");
+    ucmd.args(&["-vin", "a", "b"])
+        .pipe_in("y\n")
```
Comment (cakebaker) [inline tests/by-util/test_mv.rs:1265]: Why do you use `pipe_in` in this test case? It seems irrelevant as the test also works without it.
```suggestion
```

Comment (sylvestre) [reply tests/by-util/test_mv.rs:1265]: because you want to test the yes and no behavior, no ? :)

Comment (cakebaker) [reply tests/by-util/test_mv.rs:1265]: Yes, of course :) However, in this specific case, there is no yes/no-behavior to test. It doesn't matter what you pipe in, the output will always be the same.
